app/src/components/generic/recordCard/RecordCard.tsx

```diff
@@ -30,24 +30,19 @@ function RecordCard({
                 {detailsElement}
             </Card.Content>
             {downloadStage === DOWNLOAD_STAGE.SUCCEEDED && (
-                <Details expander open className="lloydgeorge_record-stage_pdf-expander">
-                    <Details.Summary
-                        style={{ display: 'inline-block' }}
-                        data-testid="view-record-bin"
-                    >
-                        View record
-                    </Details.Summary>
+                <div className="lloydgeorge_record-stage_pdf-expander">
                     <button
                         className="lloydgeorge_record-stage_pdf-expander-button link-button clickable"
                         data-testid="full-screen-btn"
+                        id="full-screen-btn"
```

Comment:
We don't use 'id' selectors, can you change to 'className'?

Reply:
Yeah sure, let me modify that now. thanks

Reply:
Done